src/platform_requirements.txt

```diff
@@ -1,2 +1,16 @@
-grpcio==1.38.0
+grpcio==1.62.2
 protobuf==3.20.3
+grpcio = "==1.62.2"
+psutil = "5.9.4"
+markupsafe = "2.1.5"
+protobuf = "3.20.3"
+cryptography = "==37.0.4"
+pyyaml = "==6.0.0"
+bcrypt = "4.1.2"
+cffi = "==1.15.1"
+lazy-object-proxy= "==1.9.0"
```

Comment:
What is this new package?

Reply:
Ditto

Reply:
:facepalm: since youre adding it, you can have the honor of figuring out what it is.

Reply:
Can you find out?

Reply:
If you mean lazy-object-proxy, this is a transitive dependency for pylint-protobuf
cffi is a transitive dependency for, among others, pyopenssl
bcrypt is a transitive dependency for fabric

Reply:
Now that I think of it, we can just get rid of lazy-object-proxy and typed-ast from dev dependencies, these do not run any appengine or clusterfuzz code, only the lint/format in butler.py, no reason to pin them

Reply:
Hmm there is a >0 chance that @jonathanmetzman  PR was actually correct, and Windows only broke because it was not running psutil >= 5.9.4. The largest version I saw in our windows init scripts was 5.7.0, which is not compatible with py 3.11.

I might be wrong, but this seems very plausible